src/slider/index.html

```diff
@@ -0,0 +1,11 @@
+<div class="gb-slider__base" mousedown="{ onClick }">
+  <div class="gb-slider__handle gb-slider__handle-lower" style="{ 'left: ' + props.defaultLower + '%' }"  mousedown="{ onMouseDown }" touchstart="{ onMouseDown }">
```

Comment:
should use `riot-style`

Reply:
also, you should be able to make it an object, should make it a little easier to maintain

`style="{ { left: props.defaultLower + '%' } }"`